### PR TITLE
Switch to Ecto 2.0 beta

### PR DESCRIPTION
**Pull request by paulcsmith**
The plan is to keep this in master and make sure everything works. Then switch to Ecto 2.0 final (once it's release) before releasing a new version of ExMachina.

### CODE REVIEW

File: lib/ex_machina/ecto.ex [hunk 1]
```diff
@@ -78,124 +78,18 @@ defmodule ExMachina.Ecto do
...
-
-    record
-    |> record_to_changeset
-    |> repo.insert!
-    |> restore_belongs_to_associations(record)
+  def save_record(repo, %{__meta__: %{__struct__: Ecto.Schema.Metadata}} = record) do
+    repo.insert!(record)
   end
-  def save_record(_, _ , record) do
+  def save_record(_ , record) do
```
Comment (joshuaclayton) [inline lib/ex_machina/ecto.ex]: extra whitespace after underscore

File: mix.exs [hunk 2]
```diff
@@ -2,7 +2,7 @@ defmodule ExMachina.Mixfile do
   use Mix.Project
 
   @project_url "https://github.com/thoughtbot/ex_machina"
-  @version "0.6.1"
+  @version "1.0.0-beta.0"
```
Comment (joshuaclayton) [inline mix.exs]: woohoo!

**Comment by joshuaclayton** [comment]
@paulcsmith one small comment, otherwise looks good

File: lib/ex_machina/ecto.ex (hunk 1 above)
Comment (paulcsmith) [inline lib/ex_machina/ecto.ex]: Good catch!

**Comment by paulcsmith** [comment]
Thanks @joshuaclayton :D

**Pull request merged**